source/core/slang-digest.h

```diff
@@ -25,5 +23,9 @@ namespace Slang
         {
```

Comment:
You can do it in the next PR, but I think just write `slang::Digest finalize() {} ` would be more convenient.

Also since `Digest` is defined in `namespace slang`, I guess it doesn't hurt to have a `using slang::Digest;` in this header.

Reply:
Change will be in the new specialization test PR